netutils/utils.go

```diff
@@ -161,8 +161,8 @@ func GenerateIfaceName(prefix string, len int) (string, error) {
 		if err != nil {
 			continue
 		}
-		if _, err := net.InterfaceByName(name); err != nil {
-			if strings.Contains(err.Error(), "no such") {
+		if _, err := netlink.LinkByName(name); err != nil {
```

Comment:
@LK4D4 If you have to do this you have to move the `GenerateIfaceName` into `utils_linux.go` and create unsupported definitions for other platforms. Otherwise it will fail to compile in windows and freebsd.